requirements.txt

```diff
@@ -1,5 +1,5 @@
-openwisp-users~=0.2.0
-openwisp-utils~=0.5.0
+openwisp-users~=0.3.0
```

Comment:
[0.4.0](https://github.com/openwisp/openwisp-users/blob/master/CHANGES.rst#version-040-2020-08-23) should be our new default